docs/victorialogs/querying/README.md

```diff
@@ -526,6 +530,10 @@ If `<end>` is missing, then it equals to the maximum timestamp across logs store
 The `<step>` arg can contain values in [the format specified here](https://docs.victoriametrics.com/victorialogs/logsql/#stats-by-time-buckets).
 If `<step>` is missing, then it equals to `1d` (one day).
 
+Note: The range endpoint
+[`/select/logsql/stats_query_range`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-range-stats) relies on `_time` for time bucketing and therefore does not allow any pipe to change or remove the `_time` field before the `| stats ...` pipe.  
+In contrast, instant stats queries can include any pipes before the `| stats ...` pipe, including pipes that modify or remove the `_time` field. This relaxation applies only to the instant endpoint.
```

Comment:
```suggestion
In contrast, queries passed to [`/select/logsql/stats_query`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-stats) can include any pipes before the `| stats ...` pipe, including pipes that modify or remove the `_time` field.
```

Nobody knows what does the `instant stats queries` mean. It is better to put a link to the docs for this query type, so curious readers could familiarize themselves.